Source/WebKit/Platform/IPC/ArgumentCoders.h

```diff
@@ -705,21 +705,20 @@ template<typename... Types> struct ArgumentCoder<std::variant<Types...>> {
 
         unsigned i = variant.index();
         encoder << i;
-        encode(encoder, std::forward<T>(variant), std::index_sequence_for<Types...> { }, i);
+        encode(encoder, std::forward<T>(variant), std::index_sequence<> { }, i);
     }
 
     template<typename Encoder, typename T, size_t... Indices>
     static void encode(Encoder& encoder, T&& variant, std::index_sequence<Indices...>, unsigned i)
     {
-        constexpr size_t Index = sizeof...(Types) - sizeof...(Indices);
-        static_assert(Index < sizeof...(Types));
-        if (Index == i) {
-            encoder << std::get<Index>(std::forward<T>(variant));
-            return;
+        constexpr size_t Index = sizeof...(Indices);
```

Comment:
IIRC this was discussed also last time: this variable could be just `index`, like other constexpr variables.

Reply:
Yeah, will change it.